OrbitGl/GpuTrack.cpp

```diff
@@ -16,24 +18,35 @@ using orbit_client_protos::TimerInfo;
 constexpr const char* kSwQueueString = "sw queue";
 constexpr const char* kHwQueueString = "hw queue";
 constexpr const char* kHwExecutionString = "hw execution";
+constexpr const char* kCmdBufferString = "command buffer";
 
 namespace orbit_gl {
 
 std::string MapGpuTimelineToTrackLabel(std::string_view timeline) {
   std::string label;
   if (timeline.rfind("gfx", 0) == 0) {
+    if (timeline.find("_marker") != std::string::npos) {
```

Comment:
Do we ourselves define the timeline strings in this case?

Reply:
Yes exactly. We are going to add append the "_marker" to the original timeline string.